tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1338,7 +1346,7 @@ def fit_pseudolabel(self, pseudo_data: pd.DataFrame, max_iter: int = 5, return_p
                 self.fit_weighted_ensemble()
 
             if return_pred_prob:
-                y_pred_proba = self.predict_proba(pseudo_data)
+                y_pred_proba = self.predict_proba(pseudo_data) if self.can_predict_proba else self.predict(pseudo_data)
```

Comment:
nit: split into if-else to have consistency vs other places?

Reply:
Will keep for now, it will be unified once we apply black in future